packages/camera/camera_android_camerax/lib/src/camera_selector.dart

```diff
@@ -44,10 +44,19 @@ class CameraSelector extends JavaObject {
   late final CameraSelectorHostApiImpl _api;
 
   /// ID for front facing lens.
-  static const int LENS_FACING_FRONT = 0;
+  ///
+  /// See https://developer.android.com/reference/androidx/camera/core/CameraSelector#LENS_FACING_FRONT().
+  static const int lensFacingFront = 0;
```

Comment:
Why do not we have a copy of lens facing unknown?

Reply:
We aren't using it now, but we will need that later on, so I added it!